src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -109,13 +114,10 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
         source_control_type = SourceControlType.github.value
 
     source_triggers = None
-    source_trigger_events = []
-    if commit_trigger_enabled:
-        source_trigger_events.append(SourceTriggerEvent.commit.value)
-    if pull_request_trigger_enabled:
-        source_trigger_events.append(SourceTriggerEvent.pullrequest.value)
-    # if source_trigger_events contains any event types we assume they are enabled
-    if source_trigger_events:
+    if commit_trigger_enabled or pull_request_trigger_enabled:
```

Comment:
If you can only have one type of trigger, it seems that rather than have a bunch of mutually exclusive flags you should just have a single enum, `--trigger` which can accept values like `Commit` or `PullRequest`.

Reply:
We would like to have these two flags for now for compatibility but in the future we are planning to add a separate command group `trigger` that allows the user to manage their triggers more cleanly than the current model.

Reply:
@tjprescott Though having a a single enum `--trigger` would be much clearer, it is also a breaking change. Do you recommend moving forward with this or is the above plan acceptable?